.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - run:
           command: apk update
       - run:
-          command: apk add --no-progress python3 py-pip curl jq git bash
+          command: apk add --no-progress python3 py-pip curl jq git bash py3-pip
```

Comment:
Interesting, when I tested on my test environment, `apk add py-pip` installed `py3-pip` automatically.
Sounds better to replace the `py-pip` with `py3-pip`, instead of appending a new argument.

Reply:
Good idea. I was trying to build this on my Server setup and was getting an error that "pip3 command was not found".

Reply:
Really interesting... I tried it on my server, too! 🤔